app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: List[Car]) -> float:
+        income = 0
+        for car in cars:
+            if self.wash_single_car(car):
+                income += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
+
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        single_income = round(
+            (car.comfort_class * (self.clean_power - car.clean_mark)
+             * self.average_rating)
+            / self.distance_from_city_center, 1)
+
+        return single_income
+
+    def wash_single_car(self, car: Car) -> bool:
+        return self.clean_power > car.clean_mark
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/216607518-ef383df3-6261-49a6-a261-c1bec3494ae8.png)
fix this, according to requirements